src/app.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='../typings/tsd.d.ts' />
```

Comment:
Yeah, I guess we can leave this as it is right now. If this files' purpose drastically changes later, we'll change it :-)